src/App.js

```diff
@@ -57,36 +78,54 @@ const App = ({
     setFormState(initialState);
     setEdit(false);
     updateNoteAction(updatedNote);
+    history.push('/');
   };
 
   return (
-    <div className="App">
-      <p>Notes</p>
-      <Form
-        formState={formState}
-        setInput={setInput}
-        createNote={onCreateNote}
-        handleSubmit={handleSubmit}
-        edit={edit}
-      />
-      <Notes
-        notes={notesArray}
-        loading={isLoading}
-        errors={errors}
-        deleteNote={onDeleteNote}
-        updateNote={onUpdateNote}
-      />
-    </div>
+    <AppContainerStyled>
+      <Naviagtion />
+      <Switch>
+        <Route path="/form">
```

Comment:
Perhaps a route that includes the note ID would be useful (for bookmarking etc).
e.g. /note/<uuid>

Reply:
This is a good idea! I'll work on that later, if I have more time :)